recipes/abseil/all/conandata.yml

```
@@ -12,6 +15,11 @@ sources:
     url: "https://github.com/abseil/abseil-cpp/archive/20200225.3.tar.gz"
     sha256: "66d4d009050f39c104b03f79bdca9d930c4964016f74bf24867a43fbdbd00d23"
 patches:
+  "20220623.0":
+    - patch_file: "patches/0003-absl-string-libm.patch"
+      base_path: "source_subfolder"
+    - patch_file: "patches/0005-has-unique-object-representations.patch"
```

Can you please look into adding https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches metadata so we can please track where these patches come from? 🙏

---

I have no idea about the 0003 but 0005 I made myself, do you want me to make a PR on https://github.com/abseil/abseil-cpp for that?

I didn't initially because the code is correct, it just doesn't work on GCC 7.2 due to a compiler bug (https://gcc.gnu.org/bugzilla/show_bug.cgi?id=83830). Maybe I could just link to that bug in the metadata?

---

Opening a PR against upstream is always strongly encouraged! We love seeing CCI contributions go back to projects ❤️ 

Yes for the metadata

```suggestion
    - patch_file: "patches/0005-has-unique-object-representations.patch"
      patch_description: "small workward to support GCC 7.2 which has a bug"
      patch_type: "portability"
      patch_source: "https://gcc.gnu.org/bugzilla/show_bug.cgi?id=83830"
```